ImplicitPackageReferenceUnitTests/ImplicitPackageReferenceUnitTests.cs

```diff
@@ -31,80 +31,83 @@ public void WhenTaskIsGivenSinglePackageTaskShouldEditJsonFileAndReturnTrue()
         {
             var implicitPacker = new AddImplicitPackageReferences(log);
             TaskItem implicitDependency = new TaskItem("Microsoft.Extensions.DependencyInjection");
+            implicitDependency.SetMetadata("TargetFramework", "netstandard2.0");
```

Comment:
The changes to these tests should be ignored.  The are actually a result of the previous PR.  In that PR, I changed it so that ImplicitPackageReference *only* runs if TargetFramework is set on the metadata.  Previously, if TargetFramework metadata was not set, then it would have ran, applying its changes for each target framework.

This is not desired behavior because on multi-targetting builds, the EXE is called multiple times, once for each target framework.  I changed the behavior, but forgot to run these unit tests afterwards.